inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -685,7 +690,8 @@ int main(int argc, char* argv[]) {
         std::cout << "Count:      " << iteration << " iterations" << std::endl;
         std::cout << "Duration:   " << double_to_string(totalDuration) << " ms" << std::endl;
         if (device_name.find("MULTI") == std::string::npos)
-            std::cout << "Latency:    " << double_to_string(latency) << " ms" << std::endl;
+            std::cout << "Latency (" << FLAGS_latency_percentile << " percentile):    "
```

Comment:
Latency is also reported in the report. There is a sense to add percentile in the report as well. Also it is a second reason run peformance validation: if ther parse reports, logs and so on that can affect them.

Reply:
I didn't understand what do you mean by adding percentile in the report. It is already in the report. Could you please clarify your comment?

Reply:
I suggest not to report any additional data (like user-set percentile in brackets) when the value is default (50). this would preserve the compatibility with existing routines that expect median

Reply:
@ivankochin  benchmark_app has a parameter `report_type` which allows dumping data to the file (`no_counters` value is enough to see).  BTW I agree with @myshevts that it is better to keep the format as-is for default behavior.